easybuild/easyconfigs/r/R-bundle/pbdBASE-02.3-0_ignore-BI_COMM_GLOBAL-on-intel.patch

```diff
@@ -0,0 +1,45 @@
+# Add support for intel compilers by removing references to BI_COMM_WORLD
```

Comment:
I understand, but the patch is wrong, in the sense that it's not whether or not the Intel compilers are used, but whether or not Intel MPI is used...